lusee/LBeam_Gauss.py

```diff
@@ -0,0 +1,47 @@
+from math import degrees
+from .LBeam import LBeam
+import numpy as np
+
+def gauss_beam(theta, theta_fwhm):
+    """
+    Creates a map-level gaussian beam centered at zero
+
+    for a gaussian, fwhm = 2.355*sigma
```

Comment:
Let's work with sigmas rather than FWHM. I know FWHM is more standard in radio literature, but (as a physicist) I hate it.